### DISCUSSION TITLE
Event dot based on type

### DISCUSSION THREAD

**Discussion by SphereSK** [gpbl/react-day-picker#2374] [Support]
H guys, i really strugling with creating rule where into calendar will be added dot defined based event (training, lecture) for particular day. any advice or example how to do it?

**Comment by gpbl** [comment]
@SphereSK any screenshot of the UI you are implementing? I bet this can be done via a `Day` [custom component](https://daypicker.dev/guides/custom-components#implementing-a-custom-component).
